.github/workflows/test.yaml

```diff
@@ -77,30 +40,35 @@ jobs:
           pytest -vv .
 
   protobuf:
-    name: "Generate & Diff"
+    name: "Generate & Diff Protobuf"
     runs-on: "ubuntu-latest"
     steps:
-      - uses: "actions/checkout@v2"
-      - uses: "actions/setup-python@v1"
+      - uses: "actions/checkout@v3"
+      - uses: "actions/setup-python@v4"
         with:
           python-version: "3.10"
       - name: "Setup Python Environment"
         run: "pip install -U pip virtualenv"
       - name: "Install Homebrew & gRPC"
         run: |
           bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+          echo "/home/linuxbrew/.linuxbrew/bin" >> $GITHUB_PATH
           brew install grpc
-      - uses: "bufbuild/buf-setup-action@v0.6.0"
+          ls /home/linuxbrew/.linuxbrew/bin
```

Comment:
Nice to have for visibility but we can remove these later if the buf gen works consistently for the next few updates

Reply:
agreed! I also left some extra flags in the `buf.gen.yaml` that hopefully will help troubleshoot things if we get in trouble again 😅